.github/workflows/rust.yml

```diff
@@ -141,7 +141,7 @@ jobs:
       - run: |
           echo "CHROME_PATH=${{ steps.setup-chrome.outputs.chrome-path }}" >> $GITHUB_ENV
       - name: Run tests (wasm)
-        run: cargo test --target wasm32-unknown-unknown -p ${{ matrix.workspace }}
+        run: cargo test --target wasm32-unknown-unknown --workspace -p ${{ matrix.workspace }}
```

Comment:
What changed compared to before that we need this?

Reply:
We added the v1 keystore as a dependency